src/osemgrep/networking/Semgrep_App.ml

```diff
@@ -67,18 +66,22 @@ let get_scan_config_from_token_async
     OutJ.scan_config option Lwt.t =
   let url = Uri.with_path !Semgrep_envvars.v.semgrep_url scan_config_route in
   let headers = [ Auth.auth_header_of_token caps#token ] in
-  let%lwt response = Http_helpers.get_async ~headers caps#network url in
+  let%lwt response = Http_helpers.get ~headers caps#network url in
   let scan_config_opt =
     match response with
-    | Error (msg, _) ->
-        Logs.debug (fun m -> m "error while retrieving scan config: %s" msg);
-        None
-    | Ok (body, _) -> (
+    | Ok { body = Ok body; _ } -> (
         try Some (OutJ.scan_config_of_string body) with
         | Yojson.Json_error msg ->
             Logs.debug (fun m ->
                 m "failed to parse body as scan_config %s: %s" msg body);
             None)
+    | Ok { body = Error err; code; _ } ->
+        Logs.debug (fun m ->
+            m "error while retrieving scan config %u: %s" code err);
```

Comment:
%u?

Reply:
unsigned decimal

Reply:
yeah just odd since it isn't for `Printf.printf` last I knew (and this is presumably just an `int`?)